apps/package.json

```diff
@@ -57,11 +57,9 @@
     "grunt-notify": "0.4.1",
     "grunt-sass": "1.0.0",
     "grunt-strip-code": "0.1.2",
-    "handlebars-runtime": "1.0.12",
```

Comment:
One data point: `grunt dev` worked for me after syncing this change locally, deleting `node_modules` and rebuilding everything.